ci/jjb_service_job.yml

```diff
@@ -0,0 +1,22 @@
+- job:
+    name: centos-container-pipeline-service-job
+    description: |
+        Managed by Jenkins Job Builder, do not edit manually!
+    node: container
+    properties:
+        - github:
+            url: https://github.com/rtnpro/container-pipeline-service
```

Comment:
We should probably be using https://github.com//CentOS/container-pipeline-service for this property